ci-services/jenkins.js

```diff
@@ -3,10 +3,18 @@ const env = process.env
 const _ = require('lodash')
```

Comment:
Leftover lodash require, either remove, or address my other comment for old line 8.